### ISSUE TITLE
JavaScript Error After Latest Update

### ISSUE THREAD

**Issue by lots0logs** [AndiDittrich/WordPress.Enlighter#55]
I'm seeing the following error on the frontend of my site after installing updates today (both WordPress and Enlighter were updated). 
```
Uncaught TypeError: window.addEvent is not a function
```

This is the line where the error occurs in the document:
```
EnlighterJS_Config={"selector":{"block":"pre.EnlighterJSRAW","inline":"code.EnlighterJSRAW"},"language":"generic","theme":"enlighter","indent":2,"hover":"hoverEnabled","showLinenumbers":true,"rawButton":true,"infoButton":true,"windowButton":true,"rawcodeDoubleclick":false,"grouping":true,"cryptex":{"enabled":false,"email":"mail@example.tld"}};window.addEvent('domready',function(){if(typeof EnlighterJS=="undefined"){return;};EnlighterJS.Util.Init(EnlighterJS_Config.selector.block,EnlighterJS_Config.selector.inline,EnlighterJS_Config);});;
```

**Comment by AndiDittrich** [comment]
Dear Dustin,

this is a common error caused by missing **MooTools** files - do you have disabled it within the backend ?
What was your previous Enlighter version ?

best regards, Andi

**Comment by AndiDittrich** [comment]
I've added an additional environment check to the startup code. it displays clear-text error messages (available in v3.4-BETA3)

**Example** (Missing MooTools JS File)
![enlighter_startup_error](https://cloud.githubusercontent.com/assets/1474107/21175143/4c149592-c1e1-11e6-8a74-865ab3e14a26.jpg)

**Comment by lots0logs** [comment]
Whoops, I lost track of this. Sorry! :grimacing:  The issue was caused by script optimizations performed by the cache plugin we're using. Not sure what changed (as it worked fine before) but I just adjusted the settings to ensure that the enlighter script is output after the mootools library. Thanks for following up even though I failed to do so. Much appreciated :+1:
